js/ControlTodoButton.js

```diff
@@ -0,0 +1,94 @@
+const viewAllListBtn = document.querySelector(".all"); 
+const viewTodoListBtn = document.querySelector(".active"); 
+const viewCompleteListBtn = document.querySelector(".completed"); 
+const filter = document.querySelector(".filters");
+
+export function initTodolistButton(){
+    filter.addEventListener("click", controlFilterButton);
+}
+
+function controlFilterButton({target}){
+    if(target.classList.contains('all')) chooseButton('all');
+    else if(target.classList.contains('active')) chooseButton('active');
+    else if(target.classList.contains('completed')) chooseButton('completed');
+}
+
+export function chooseButton(button){
+    if(button === 'all') viewAll();
+    else if(button === 'active') viewTodo();
+    else if(button === 'completed') viewDone();
+}
```

Comment:
```suggestion
export function chooseButton(button){
  const  buttonList = {
    all: viewAll,
    active: viewTodo,
    completed: viewDone,
  }
  return  buttonList[button]()
}
```

지난 주차 미션 피드백에 있던 object literal을 사용하면 더 깔끔한 코드 작성이 가능할거 같습니다!

Reply:
이런 방식으로도 함수를 실행 가능하다는게 신기하네요.. 피드백을 보고 생각만 했지 이렇게 활용할 수 있을거라고는 전혀 몰랐습니다.
review#4 에 반영하여 수정하였습니다. 감사합니다!